service/codegen/src/main/java/io/helidon/service/codegen/ServiceDescriptorCodegen.java

```diff
@@ -1644,6 +1656,8 @@ private void createInstantiateBody(TypeName serviceType,
                 .map(ParamDefinition::ipParamName)
                 .collect(Collectors.joining(", "));
 
+        method.addContentLine("LOGGER.log(System.Logger.Level.DEBUG, \"instantiate (weight = \" + weight() + \", run level = \" + runLevel().orElse(null) + \")\");");
```

Comment:
This should be done slightly differently to avoid possible issues with name conflicts and imports.
Also when using any parameters, the log statement should be guarded by `LOGGER.isLoggable()`

```java
method.addContent("LOGGER.log(")
                .addContent(System.Logger.Level.class)
                .addContentLine(".DEBUG, \"Instantiate (weight = \" + weight() + \", run level....")
```